### ISSUE TITLE
Inventory is not available until admin visits the PDP

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#961]
### Expected behavior
Imported inventory should be available before the first user checks out

### Actual Behavior
Imported inventory is not inserted into the Inventory collection until an admin visits the PDP page (other actions might trigger this too, that's just what I stumbled upon). If you don't do this, then all orders are placed into backorder because there is no available inventory.

### Steps to Reproduce the Behavior
* do `./reaction reset && reaction`
* place an order
* Observe the following messages:
```
02:05:06.354Z  INFO Reaction: totalRequiredQty 1
02:05:06.354Z  INFO Reaction: availableInventoryQty 0
02:05:06.354Z  INFO Reaction: no inventory found, create 1 backorder
02:05:06.358Z  INFO Reaction: created 1 backorder records for product BCTMZ6HTxFSppJESk, variant CJoRBm9vRrorc9mxZ
```

**Comment by zenweasel** [comment]
Tested this with version 0.11.1 and it occurs as well.

**Comment by zenweasel** [comment]
Still happening in 1.3.1

**Comment by zenweasel** [comment]
Closed via #1198
